tests/test_questions.py

```diff
@@ -0,0 +1,67 @@
+# Tests for questions
+
+import unittest
+import json
+
+from app import create_app
+from app.v1.models import Question
+
+
+class TestQuestions(unittest.TestCase):
+    """Class to test for questions"""
+
+    def setUp(self):
+        """Method to create app and set up test client"""
+        self.app = create_app("testing")
+        self.app = self.app.test_client()
+
+    def test_user_can_post_question(self):
+        """Method to test if a new question can be added"""
+        response = self.add_question()
+        self.assertEqual(response.status_code, 201)
```

Comment:
Assertions should be done for response messages also.

Reply:
Please note that this applies for other test cases that have been defined in a similar manner.

Reply:
Done. Tested response messages for all post requests.